bin/yb-ctl

```diff
@@ -1863,6 +1897,16 @@ class ClusterControl:
 
     def run(self):
         self.args = self.parser.parse_args()
+        # TODO: This is required for `yb-ctl --FLAGS CMD` syntax to work. Delete this when we
+        # update the docs so we only support `yb-ctl CMD --FLAGS` syntax.
+        for flag in dir(self.args):
+            if "_dest" in flag:
+                real_flag = flag[:-5]
+                parent_arg = getattr(self.args, flag, None)
+                child_arg = getattr(self.args, real_flag, None)
+                print(str(real_flag) + ": " + str(child_arg))
```

Comment:
Log instead of print. Also, do this only in verbose mode.
It's odd to have this printed as part of cluster creation command.